src/customJs/commonProperties/utils.test.js

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { formatUrl } from './utils';
+
+describe(formatUrl.name, () => {
+  it.each([
+    { urlWithoutFormat: 'just_a_word', urlExpected: 'just_a_word' },
+    { urlWithoutFormat: '/slash.com', urlExpected: '/slash.com' },
+    {
+      urlWithoutFormat: '/slash_without_dot',
+      urlExpected: '/slash_without_dot',
+    },
+    { urlWithoutFormat: ':dot.com', urlExpected: ':dot.com' },
+    { urlWithoutFormat: 'dot.com', urlExpected: 'https://dot.com' },
+    { urlWithoutFormat: 'dot.com?', urlExpected: 'https://dot.com?' },
+    { urlWithoutFormat: "dot.com'", urlExpected: "https://dot.com'" },
+    { urlWithoutFormat: 'dot.cooooooom', urlExpected: 'https://dot.cooooooom' },
+    { urlWithoutFormat: "'url", urlExpected: "'url" },
+    { urlWithoutFormat: "'url.com", urlExpected: "'url.com" },
+    { urlWithoutFormat: 'https://url.com', urlExpected: 'https://url.com' },
```

Comment:
I would add some examples to validate if we are following current Unlayer behavior:

```suggestion
    { urlWithoutFormat: 'https://url.com', urlExpected: 'https://url.com' },
    { urlWithoutFormat: 'abc cde', urlExpected: 'abc cde'},
    { urlWithoutFormat: '   abc cde   ', urlExpected: 'abc cde'},
    { urlWithoutFormat: '   a.b   ', urlExpected: 'https://a.b'},
```

![image](https://user-images.githubusercontent.com/1157864/211074396-77701efd-cb36-44ec-a7c0-f174c9a7926a.png)